src/App.js

```diff
@@ -9,7 +10,9 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <>
+    <Article article={article} />
+  </>
```

Comment:
```suggestion
  <Article {...article} />
```
Лучше через спред оператор